### ISSUE TITLE
Import declaration conflicts with local declaration of 'HotTableProps'.

### ISSUE THREAD

**Issue by ImagineZzf** [handsontable/vue-handsontable-official#145]
### Description
Before I reinstall the project, everything is normal.
Now typescript error

```js
Error in .../@handsontable/vue/types.d.ts(4,10):
4:10 Import declaration conflicts with local declaration of 'HotTableProps'.
    2 | import Vue, { VNode } from 'vue';
    3 | import { ThisTypedComponentOptionsWithRecordProps } from 'vue/types/options';
  > 4 | import { HotTableProps } from './types';
      |          ^
    5 | export interface HotTableData {
    6 |     __internalEdit: boolean;
    7 |     hotInstance?: Handsontable;
```

```js
Import declaration conflicts with local declaration of 'HotTableProps'.
```

I found that the version I installed was 4.0.0, but It's totally different in node_modules, why ?

### Your environment
* Vue wrapper version: 2.6.10
* Handsontable version: 7.0.3
* @handsontable/vue version: 4.0.0

**Comment by thinkAmi** [comment]
I had the same error message.

In my case, use TypeScript version 3.7.3.

I have fixed it by downgrading TypeScript from 3.7.3 to 3.6.4.

**Comment by AMBudnik** [comment]
Thank you for sharing this case. We need to investigate it further.

**Comment by dlkulp** [comment]
Also have this problem, don't want to have to update the rest of my project to remove TS 3.7.X features if there's any way to avoid it.

Seems to be due to this change in TS: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#local-and-imported-type-declarations-now-conflict

Looks like a quick solution is to rename the duplicate imported token like so: 
`import { module as otherModuleName } from "./module"`

Using: 
* @handsontable/vue @4.1.1  
* handsontable @7.3.0
* vue @2.6.10
* typescript @3.7.3

**Comment by zero530** [comment]
Also have this problem, but can fix it.I have comments below code and re-build, it successfully.
import { HotTableProps } from './types';

**Comment by haexhub** [comment]
In your  types.d.ts is following error:

_@handsometable/vue/types.d.ts_
```
import { HotTableProps } from './types';

export interface HotTableProps extends Handsontable.GridSettings {
    id?: string;
    settings?: Handsontable.DefaultSettings;
    wrapperRendererCacheSize?: number;
}
```

Your are importing a type and exporting  a different type with the same name. Can you please correct this?

**Comment by wojciechczerniak** [comment]
Fixed and released in v5.0.1
